connect-base/2.1/docker-entrypoint.sh

```diff
@@ -94,6 +95,21 @@ else
     done
 fi
 
+if [[ "${ENABLE_DEBEZIUM_KC_REST_EXTENSION}" == "true" && ! -f "$EXTERNAL_LIBS_DIR" && -d "$EXTERNAL_LIBS_DIR/debezium-connect-rest-extension" ]] ; then
```

Comment:
@dude0001 Looking at the script, would it be possible to refactor it a bit - using function maybe as it seems doing Apicurio, rest-extension and scripting is same in the functionality

Reply:
The debezium-scripting and apicurio libraries are being added as additional resources to each Debezium Connector. The Debezium REST API Extension needs to be deployed as its own Kafka Connect plugin is my understanding, and not a resource for Debezium Connectors. If that statement is true, Debezium REST API Extension needs its own pattern.

I refactored out a function with shared logic for enabling additional resources to all Debezium Connectors, and debezium-scripting and apicurio use this now.

I created another function that is specific to the REST API Extension for now, which could be tweaked more in the future to enable other optional true Kafka Connect plugins.

I was able to test to be working for the REST API Extension and did some regression for debezium-scripting and apicurio, and it looks to be configuring everything as expected. Please take a look, I would love additional feedback if I need to take another pass at this.